drivers/clock_control/clock_stm32l4_l5_wb_wl.c

```diff
@@ -40,27 +62,35 @@ int config_pll_sysclock(void)
 	}
 #endif /* PWR_CR5_R1MODE */
 
-	pll_n = STM32_PLL_N_MULTIPLIER;
-	pll_m = pllm(STM32_PLL_M_DIVISOR);
-	pll_r = pllr(STM32_PLL_R_DIVISOR);
-
-	/* Configure PLL source */
-	if (IS_ENABLED(STM32_PLL_SRC_HSI)) {
-		pll_source = LL_RCC_PLLSOURCE_HSI;
-	} else if (IS_ENABLED(STM32_PLL_SRC_HSE)) {
-		pll_source = LL_RCC_PLLSOURCE_HSE;
-	} else if (IS_ENABLED(STM32_PLL_SRC_MSI)) {
-		pll_source = LL_RCC_PLLSOURCE_MSI;
-	} else {
-		return -ENOTSUP;
-	}
-
-	LL_RCC_PLL_ConfigDomain_SYS(pll_source, pll_m, pll_n, pll_r);
+	LL_RCC_PLL_ConfigDomain_SYS(pll_source,
+				    pllm(STM32_PLL_M_DIVISOR),
+				    STM32_PLL_N_MULTIPLIER,
+				    pllr(STM32_PLL_R_DIVISOR));
 
 	LL_RCC_PLL_EnableDomain_SYS();
 
 	return 0;
 }
+
+/**
+ * @brief Return pllout frequency
+ */
+__unused
+uint32_t get_pllout_frequency(void)
+{
+	uint32_t pll_source;
+
+	pll_source = get_pll_source();
+	if (pll_source < 0) {
```

Comment:
`get_pll_source()` returns `uint32_t`.
Therefore, `if (pll_source < 0)` is always `false`.